src/util.js

```diff
@@ -108,3 +110,33 @@ exports.getRange = function getRange(from, to) {
     }
     return arr;
 };
+
+exports.sumByRow = function sumByRow(matrix) {
+    var sum = Matrix.Matrix.zeros(matrix.rows, 1);
+    for (var i = 0; i < matrix.rows; ++i) {
+        for (var j = 0; j < matrix.columns; ++j) {
+            sum[i][0] += matrix[i][j];
```

Comment:
Please use `sum.set()` and `matrix.get()`.
We abstract out the fact that Matrix is a 2D array to allow different kinds of storages (like all the views)